.prettierrc.yml

```diff
@@ -0,0 +1,7 @@
+arrowParens: always
```

Comment:
Can you add `printWidth: 100` from the JS pr

Reply:
Done!